.azure/gpu-tests-pytorch.yml

```diff
@@ -51,10 +51,6 @@ jobs:
     cancelTimeoutInMinutes: "2"
     strategy:
       matrix:
-        'PyTorch & strategies':  # this uses torch 1.12 as not all strategies support 1.13 yet
```

Comment:
why remove this from the matrix?
we should still test with deepspeed, it is not external

Reply:
in such case we need to add `deepspeed` to all runs

Reply:
Adding deepspeed to all runs is what I did